queries.sql

```diff
@@ -12,4 +12,35 @@ SELECT * FROM animals WHERE neutered IS TRUE;
 
 SELECT * FROM animals WHERE name != 'Gabumon';
 
-SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
\ No newline at end of file
+SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
+
+-- update animals
+-- 1
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+SELECT * FROM animals;
+ROLLBACK;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon' WHERE species='';
+COMMIT;
+-- 2
+BEGIN;
+DELETE FROM animals;
+ROLLBACK;
+-- 3
+BEGIN;
+DELETE FROM animals WHERE date_of_birth > '2022-1-1';
+SAVEPOINT s1;
+UPDATE animals SET weight_kg = weight_kg*-1;
+ROLLBACK TO s1;
+UPDATE animals SET weight_kg = weight_kg*-1 WHERE weight_kg < 0;
+COMMIT;
+-- 4
+SELECT count(*) FROM animals;
+SELECT count(*) FROM animals WHERE escape_attempts >= 1;
```

Comment:
- [ ] Kindly the question is: `How many animals have never tried to escape?` please the query is not correct, Kindly fix this 🙏